app/views/console/storage/bucket.phtml

```diff
@@ -0,0 +1,515 @@
+<?php
+$home = $this->getParam('home', '');
+$fileLimit = $this->getParam('fileLimit', 0);
+$fileLimitHuman = $this->getParam('fileLimitHuman', 0);
+$bucketPermissions = $this->getParam('bucketPermissions', null);
+$fileCreatePermissions = $this->getParam('fileCreatePermissions', null);
+$fileUpdatePermissions = $this->getParam('fileUpdatePermissions', null);
+?>
+
+<div
+    data-service="storage.getBucket"
```

Comment:
We need to remove this file. We no longer use it. You need to add the compression option here
https://github.com/appwrite/console/blob/72b3f26edb802517b8d1c9d02900dbb99da58d38/src/routes/console/project-%5Bproject%5D/storage/bucket-%5Bbucket%5D/settings/+page.svelte#L49